convert.py

```diff
@@ -127,8 +149,51 @@ def convert_code(text, file):
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum Code {""", file=file)
-    display = handle_enum_entries(text, file)
-    print("\n}", file=file)
+    display = parse(text)
+
+    for i in range(13, 25):
+        display.append([
+            'F{}'.format(i),
+            '    /// <code class="keycap">F{}</code>\n'.format(i),
+            []
+        ])
+
+    for chromium_only in [
```

Comment:
can you put this list in a separate variable `chromium_key_codes`? Multi-line loop conditions are a bit uncommon.